src/TicTacToe.Console/GameConfiguration/GameConfigurationService.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using iTechArt.TicTacToe.Foundation.Figures;
+using iTechArt.TicTacToe.Foundation.Interfaces;
+using TicTacToe.Console.Extensions;
+using TicTacToe.Console.Interfaces;
+
+namespace TicTacToe.Console.GameConfiguration
+{
+    public class GameConfigurationService : IGameConfigurationService
+    {
+        private readonly IGameConfigurationFactory _configurationFactory;
+        private readonly IPlayersRegistrationService _registrationService;
+        private readonly IConsole _console;
+        private readonly IConsoleInputProvider _consoleInput;
+
+        private IList<IPlayer> _players;
```

Comment:
Please, find another way to share players list between games, without introducing state to this service.